bin/sozo/src/commands/model.rs

```diff
@@ -16,8 +16,8 @@ pub struct ModelArgs {
 
 #[derive(Debug, Subcommand)]
 pub enum ModelCommands {
-    #[command(about = "Retrieve the class hash of a model")]
-    ClassHash {
+    #[command(about = "Retrieve the contract address of a model")]
```

Comment:
Hm I thought that we would continue using ClassHash here and the contract address / deploying models is just for registration?

Reply:
Fixed in 0456f4c58cf39141912d5551fc4790dc2647f978.